searx_extra/update/update_engine_descriptions.py

```diff
@@ -3,16 +3,19 @@
 import sys
 import json
 from urllib.parse import quote, urlparse
-import detect_language
 from lxml.html import fromstring
+import cld3
 
-from searx.engines.wikidata import send_wikidata_query
+from searx.engines import wikidata
+from searx.engines import set_loggers
```

Comment:
Lines 9 and 10 --> `from searx.engines import wikidata, set_loggers`

Reply:
sure .. 🤦🏻  .. done!